input/resources/au-erequesting-server.xml

```diff
@@ -349,5 +349,176 @@
         </extension>
       </referencePolicy> -->
     </resource>
+    <!-- TASK -->
+    <resource>
+      <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+        <valueCode value="SHALL"/>
+      </extension>
+      <type value="Task"/>
+      <supportedProfile value="http://hl7.org.au/fhir/ereq/StructureDefinition/au-erequesting-task">
+        <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+          <valueCode value="SHALL"/>
+        </extension>
+      </supportedProfile>
+      <documentation value="Servers **SHALL** support the Task resource, the AU eRequesting profile, and the conformance expectations for the Task resource.&#xA;&#xA;The Server **SHALL** support the `_include` parameters for `Task:patient`, `Task:requester`, `Task:owner`, `Task:focus`."/>
```

Comment:
'parameters' should be singular 'parameter'.

Reply:
Amended, thanks.